src/toil_lib/tools/preprocessing.py

```diff
@@ -1,12 +1,14 @@
 import os
 
+from toil.job import PromisedRequirement as PrR
```

Comment:
I'll buy you two tacos if you import this by its regular name. I'm worried about readability and consistency across pipelines. For example, I just saw that @jvivian imports this as PReq. Alternatively we should collectively settle on an alternative name.